modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-customer-portal/extra/remote-app/src/common/services/liferay/rest/raysource/LicenseKeys.js

```diff
@@ -50,3 +50,23 @@ export async function getDevelopmentLicenseKey(
 
 	return response;
 }
+
+export async function getAllActivationKeys(
+	accountKey,
+	licenseKeyDownloadURL,
+	sessionId
+) {
+	// eslint-disable-next-line @liferay/portal/no-global-fetch
+	const response = await fetch(
+		`${licenseKeyDownloadURL}/accounts/${accountKey}/license-keys/export`,
+
+		{
+			headers: {
+				'Content-Type': 'application/*',
```

Comment:
You can remove this line, this comes by default in fetch

Reply:
Of course! Thank you for this Iago!